travis/build_info.py

```diff
@@ -228,6 +295,8 @@ def main(raw_args):
     logging.getLogger(__name__).setLevel(args.log_level.upper())
     if args.task_class.upper() == 'JOB':
         get_job_counts(org=args.org)
+    elif args.task_class.upper() == 'DURATION':
+        get_average_duration_org(org=args.org)
```

Comment:
Using `click` might make this whole thing cleaner. You could define a `click.group` for the `build_info` script, and then decorate each report function with `click.command`, and it would wire them together for you.